### ISSUE TITLE
Tremendously slow

### ISSUE THREAD

**Issue by ghost** [soimort/translate-shell#364]
Hello there, I really don't want to be the guy pointing out a performance issue, since it might aswell reside on my networks part. But I can't believe that my network is as bad as shown in below example. Especially since the real value typically evaluates from adding user & sys together (?)

```
chris:~$ time trans :de test
test
/test/
...
(more output)
...
test
    Prüfung, Test, testen

real	1m15.550s
user    0m0.238s
sys	0m0.137s
chris:~$ 
```

I was really hoping to get some help on potential causes on that performance lack, since I would love to use this CLI in the future.

I'm using the brew installation on MacOS Catalina (10.15.6) within bash (5.0.18). 
I'm also using tor as a proxy in my Firefox (default browser), what might interfere network performance (?)

Feel free to close this issue, if you really demand me on believing that it might just be a network issue on my part, which would be way to vague to troubleshoot.

**Comment by soimort** [comment]
1. Please post the output of `trans -V`.
2. Your issue seems be related to: #287 (which should be fixed in version 0.9.6.10 and above). You may try disabling IPv6 on your system or using `trans` with option `-4` to see if the slowness persists.

**Comment by ghost** [comment]
Oh wow, option `-4` indeed solved it. Thank you so much for the quick response! I'm going to close this issue.
